www/src/pgmbio.entry.jsx

```diff
@@ -22,37 +24,51 @@ window.jQuery = $;
 window.$ = $;
 var materialize = require('./lib/materialize.min.js')
 var graphvis = require('./lib/graphvis.js');
-var autobahn = require('autobahn');
+var autobahn = require("autobahn");
+var when = require("when");
 
 const wsuri = (document.location.origin == "file://") ?
     "ws://127.0.0.1/ws" :
-    (document.location.protocol === "http:" ? "wss:" : "wss:") + "//127.0.0.1/ws";
+    (document.location.protocol === "http:" ? "wss:" : "wss:") + "//127.0.0.1:443/ws";
```

Comment:
this make the beginning "wss:" either way. And I think the default port will be 443 for "wss" so it is probably not required.